proofs/signatures/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ set(core_signature_files
     th_bv_rewrites.plf
     th_real.plf
     th_int.plf
-    th_lra.plf
+    th_lira.plf
```

Comment:
I wonder if `th_lra.plf` should be now deleted? or maybe safer to write a comment at its first line?

Reply:
I generally try to avoid checking commented-out code into version control, preferring to let the history keep it.

I'll remove `th_lra.plf` in a subsequent PR.